packages/office-ui-fabric-react/src/components/Modal/examples/Modal.Basic.Example.tsx

```diff
@@ -64,22 +87,32 @@ export class ModalBasicExample extends React.Component<{}, IModalBasicExampleSta
   public render(): JSX.Element {
     return (
       <div>
-        <Checkbox label="Is draggable" onChange={this._toggleDraggable} checked={this.state.isDraggable} />
+        <Checkbox
+          styles={{ root: { marginBottom: '20px' } }}
+          label="Is draggable"
+          onChange={this._toggleDraggable}
+          checked={this.state.isDraggable}
+        />
         <DefaultButton secondaryText="Opens the Sample Modal" onClick={this._showModal} text="Open Modal" />
         <Modal
           titleAriaId={this._titleId}
           subtitleAriaId={this._subtitleId}
           isOpen={this.state.showModal}
           onDismiss={this._closeModal}
           isBlocking={false}
-          containerClassName={styles.container}
+          containerClassName={modalStyles.container}
           dragOptions={this.state.isDraggable ? this._dragOptions : undefined}
         >
-          <div className={styles.header}>
+          <div className={modalStyles.header}>
```

Comment:
This is still a bit weird. It's called `modalStyles` but it's not used for the `Modal`? Bit of a nitpick, but maybe it would be better as `styles` or `contentStyles` to minimize confusion.

Reply:
You bet!